packages/arb-token-bridge-ui/src/components/App/useSyncConnectedChainToQueryParams.ts

```diff
@@ -28,11 +36,12 @@ export function useSyncConnectedChainToQueryParams() {
     const { sourceChainId: sourceChain, destinationChainId: destinationChain } =
       sanitizeQueryParams({
         sourceChainId: chain.id,
-        destinationChainId: undefined
+        destinationChainId: undefined,
+        disableTransfersToNonArbitrumChains
```

Comment:
Up for a discussion, this is just an idea. What if instead of passing a boolean flag everywhere into functions (which personally I'm not a big fan of as they aren't scalable and serve a single purpose), we did something like this:
```typescript
sanitizeQueryParams({
  sourceChainId: chain.id,
  destinationChainId: undefined,
  blockedSourceChainIds: [],
  blockedDestinationChainIds: getNonArbitrumChainIds()
})
```
then similarly in other functions:
```typescript
getDestinationChainIds({
  chainId,
  blockedChainIds: getNonArbitrumChainIds()
})
```